packages/lit-element/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "lit-element",
+  "version": "3.0.0-pre.0",
+  "description": "A simple base class for creating fast, lightweight web components",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "Polymer/lit-element",
+  "type": "module",
+  "main": "lit-element.js",
+  "module": "lit-element.js",
+  "directories": {
+    "test": "test"
+  },
+  "files": [
+    "/lib/",
+    "/src/",
+    "!/src/demo/",
+    "!/src/test/",
+    "/lit-element.d.ts",
+    "/lit-element.d.ts.map",
+    "/lit-element.js",
+    "/lit-element.js.map",
+    "/ts3.4/"
+  ],
+  "scripts": {
+    "build": "tsc && rm -rf ./ts3.4 && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
+    "build:watch": "tsc --watch",
+    "build:babel-test": "babel src/test/lib/decorators_test.ts --out-file test/lib/decorators-babel_test.js",
+    "gen-docs": "typedoc src/lit-element.ts src/lib/updating-element.ts",
+    "test": "npm run build && npm run build:babel-test && wct",
+    "quicktest": "wct -l chrome -p",
+    "checksize": "rollup -c ; rm lit-element.bundled.js",
+    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
+    "lint": "tslint --project ./",
+    "prepublishOnly": "node check-version-tracker.cjs && npm run lint && npm test",
+    "prepare": "npm run build",
+    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install",
+    "publish-dev": "npm test && VERSION=${npm_package_version%-*}-dev.`git rev-parse --short HEAD` && npm version --no-git-tag-version $VERSION && npm publish --tag dev",
+    "release": "np --any-branch --yolo"
+  },
+  "author": "The Polymer Authors",
+  "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-proposal-decorators": "^7.2.3",
+    "@babel/plugin-transform-typescript": "^7.2.0",
+    "@types/chai": "^4.0.1",
+    "@types/mocha": "^5.2.4",
+    "@webcomponents/shadycss": "^1.8.0",
+    "@webcomponents/webcomponentsjs": "^2.2.3",
+    "chai": "^4.0.2",
+    "clang-format": "^1.2.4",
+    "downlevel-dts": "^0.6.0",
+    "lit-element-benchmarks": "^0.1.0",
+    "mocha": "^6.2.2",
+    "np": "^5.2.1",
+    "rollup": "^1.26.0",
+    "rollup-plugin-filesize": "^6.2.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-terser": "^5.1.0",
+    "tachometer": "^0.4.16",
+    "tslint": "^5.20.1",
+    "typedoc": "^0.17.0-3",
+    "typescript": "~3.8.2",
+    "uglify-es": "^3.3.9",
+    "wct-mocha": "^1.0.0",
+    "web-component-tester": "^6.9.2"
+  },
+  "typings": "lit-element.d.ts",
+  "typesVersions": {
```

Comment:
Let's remove this. We'll drop support for ts3.4 in the next version.

Reply:
Done.